### PR TITLE
[CN-1082] Update MC ingress in external access page

### PR DESCRIPTION
**Pull request by semihbkgr**
Changes:
- add `path` section in example
- add warning about complex paths

### CODE REVIEW

File: docs/modules/ROOT/pages/management-center-external-access.adoc [hunk 1]
```diff
@@ -26,28 +26,30 @@ External connectivity defines how to expose Management Center:
 
 |===
 
-=== Example Management Center Ingress Configuration
+=== Example Management Center LoadBalancer Configuration
 
 The example configuration below does the following:
 
-- Expose Management Center using Ingress defined on the https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress].
+- Expose Management Center using a dedicated load balancer for your Management Center
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/management-center-external-access.adoc:33]:
```suggestion
- Expose Management Center using a dedicated load balancer
```

File: docs/modules/ROOT/pages/management-center-external-access.adoc [hunk 2]
```diff
@@ -26,28 +26,30 @@ External connectivity defines how to expose Management Center:
...
+include::ROOT:example$/management-center-loadbalancer.yaml[]
 ----
 
-=== Example Management Center LoadBalancer Configuration
+=== Example Management Center Ingress Configuration
 
 The example configuration below does the following:
 
-- Expose Management Center using a dedicated load balancer for your Management Center
+- Expose Management Center using Ingress defined on the https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress].
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/management-center-external-access.adoc:44]:
```suggestion
- Expose Management Center using Ingress defined on https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress].
```

File: docs/modules/ROOT/pages/management-center-external-access.adoc [hunk 3]
```diff
@@ -26,28 +26,30 @@ External connectivity defines how to expose Management Center:
...
 
 [source,yaml,subs="attributes+"]
 ----
-include::ROOT:example$/management-center-loadbalancer.yaml[]
+include::ROOT:example$/management-center-ingress.yaml[]
 ----
 
+WARNING: The paths, consisting of multiple segments (e.g., "/hazelcast/mancenter"), are supported as of Management Center version 5.4.0.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/management-center-external-access.adoc:51]:
```suggestion
WARNING: Paths that consist of multiple segments, such as _/hazelcast/mancenter_, are supported from Management Center version 5.4.0.
```

Comment (rebekah-lawrence) [reply docs/modules/ROOT/pages/management-center-external-access.adoc:51]: Why is this a warning? Is anything broken by this change? What are the implications of this change?

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/examples/management-center-ingress.yaml [hunk 4]
```diff
@@ -10,8 +10,9 @@ spec:
     ingress:
       hostname: "example.hostname"
       ingressClassName: nginx
+      path: "/mancenter"
```
[the remark this answers is not in the document]
Comment (semihbkgr) [reply docs/modules/ROOT/examples/management-center-ingress.yaml:13]: Well, I didn't see the sub-filed in the table.
https://docs.hazelcast.com/operator/latest-snapshot/management-center-external-access#configuring-hazelcast-external-access

File: docs/modules/ROOT/examples/management-center-ingress.yaml [hunk 5]
```diff
@@ -10,8 +10,9 @@ spec:
...
       annotations:
-        nginx.ingress.kubernetes.io/rewrite-target: /
+        nginx.ingress.kubernetes.io/enable-opentelemetry: "true"
```
[the remark this answers is not in the document]
Comment (semihbkgr) [reply docs/modules/ROOT/examples/management-center-ingress.yaml:15]: I wanted to make it like full-yaml by utilizing all available fields.

File: docs/modules/ROOT/pages/management-center-external-access.adoc (hunk 3 above)
Comment (semihbkgr) [reply docs/modules/ROOT/pages/management-center-external-access.adoc:51]: The complex paths like `/hazelcast/mancenter` will be supported after 5.4.0.

What is the best label for this message?

Comment (rebekah-lawrence) [reply docs/modules/ROOT/pages/management-center-external-access.adoc:51]: NOTE: should be sufficient

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Pull request merged**